lib/Dancer2.pm

```diff
@@ -85,7 +85,7 @@ sub import {
     $final_args{dsl} ||= $config_dsl;
 
     # load the DSL, defaulting to Dancer2::Core::DSL
-    load_class( $final_args{dsl} );
+    use_module( $final_args{'dsl'} );
     my $dsl = $final_args{dsl}->new( app => $app );
     $dsl->export_symbols_to( $caller, \%final_args );
```

Comment:
This could be chained:

`my $dsl = use_module(  $final_args{'dsl'} )->new( app => $app );`